instrumentation/graphql/test/instrumentation/graphql/tracers/graphql_tracer_test.rb

```diff
@@ -0,0 +1,292 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+require 'test_helper'
+
+require_relative '../../../../lib/opentelemetry/instrumentation/graphql'
+require_relative '../../../../lib/opentelemetry/instrumentation/graphql/tracers/graphql_tracer'
```

Comment:
Aren't these already required in the `test_helper`?

Reply:
They are not.

```
graphql % be appraisal rake test
>> BUNDLE_GEMFILE=/Users/robert/src/github.com/open-telemetry/opentelemetry-ruby-contrib/instrumentation/graphql/gemfiles/graphql_1.13.gemfile bundle exec rake test
/Users/robert/src/github.com/open-telemetry/opentelemetry-ruby-contrib/instrumentation/graphql/test/instrumentation/graphql/tracers/graphql_trace_test.rb:12:in `<top (required)>': uninitialized constant OpenTelemetry::Instrumentation::GraphQL::Tracers (NameError)

describe OpenTelemetry::Instrumentation::GraphQL::Tracers::GraphQLTrace do
                                                ^^^^^^^^^
Did you mean?  OpenTelemetry::Trace
```

Reply:
Interesting